lib/Github/Api/Repository/Actions/Workflows.php

```diff
@@ -24,30 +24,48 @@ public function all(string $username, string $repository, array $parameters = []
     }
 
     /**
-     * @link https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#get-a-workflow
+     * @link https://docs.github.com/en/rest/reference/actions#get-a-workflow
      *
      * @param string $username
      * @param string $repository
-     * @param int    $workflowId
+     * @param string $workflow
      *
      * @return array
      */
-    public function show(string $username, string $repository, int $workflowId)
+    public function show(string $username, string $repository, string $workflow)
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/actions/workflows/'.$workflowId);
+        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/actions/workflows/'.rawurlencode($workflow));
     }
 
     /**
-     * @link https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#get-workflow-usage
+     * @link https://docs.github.com/en/rest/reference/actions#get-workflow-usage
      *
      * @param string $username
      * @param string $repository
-     * @param int    $workflowId
+     * @param string $workflow
      *
      * @return array|string
      */
-    public function usage(string $username, string $repository, int $workflowId)
+    public function usage(string $username, string $repository, string $workflow)
     {
-        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/actions/workflows/'.$workflowId.'/timing');
+        return $this->get('/repos/'.rawurlencode($username).'/'.rawurlencode($repository).'/actions/workflows/'.rawurlencode($workflow).'/timing');
+    }
+
+    /**
+     * @link https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event
+     *
+     * @param string $username
+     * @param string $repository
+     * @param string $workflow
+     * @param string $ref
+     * @param array  $inputs
+     *
+     * @return array|string empty
+     */
+    public function dispatches(string $username, string $repository, string $workflow, string $ref, array $inputs = null)
```

Comment:
Maybe use the same approach here for the `$workflow` parameter, to allow both int and string. When bumping php versions we can typehint this with `string|int`.